src/main.rs

```diff
@@ -115,6 +115,7 @@ const CMDMAP: u32 = (1 << SerprogCommand::Nop as u32)
     | (1 << SerprogCommand::QCmdMap as u32)
     | (1 << SerprogCommand::QPgmName as u32)
     | (1 << SerprogCommand::QSerBuf as u32)
+    | (1 << SerprogCommand::QWrNMaxLen as u32)
```

Comment:
I think you also want to add QRdNMaxLen into this mask, no?

Reply:
Done